### PR TITLE
update xor desc

### PR DESCRIPTION
**Pull request by wanqingying**

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/operators/bitwise_xor/index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ slug: Web/JavaScript/Reference/Operators/Bitwise_XOR
 
 {{jsSidebar("Operators")}}
 
-**按位异或**（**`^`**）运算符在其中任意一个操作数对应的二进制位为 `1` 时，该位的结果值为 `1`。
+**按位异或**（**`^`**）运算符在其中任意一个操作数对应的二进制位不同时，该位的结果值为 `1`。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/operators/bitwise_xor/index.md:8]: two suggestions:

## translation of [english doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_XOR)

```suggestion
**按位异或**（**`^`**）运算符在其中任意一个操作数对应的二进制位为 `1`（但不是两个操作数的对应二进制位均为 `1`）时，该位的结果值为 `1`。
```

## improvement

```suggestion
**按位异或**（**`^`**）运算符在两个操作数对应的二进制位不同时，该位的结果值为 `1`。
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

**Comment by yin1999** [comment]
Blocked by: mdn/content#23086

File: files/zh-cn/web/javascript/reference/operators/bitwise_xor/index.md (hunk 1 above)
Comment (Josh-Cena) [reply files/zh-cn/web/javascript/reference/operators/bitwise_xor/index.md:8]: Another version that is in line with English version:

```suggestion
**按位异或**（**`^`**）运算符在两个操作数对应位有且仅有一个为 `1` 时，该位的结果值为 `1`。
```

**Review by yin1999 — APPROVED** [review]

**Comment by yin1999** [comment]
Thanks for your suggestions @Josh-Cena.

And thanks for catching the wrong translation @wanqingying. Welcome to the community 🎉

**Pull request merged**